src/App.js

```diff
@@ -21,8 +24,11 @@ function App() {
           <Route path="/ProductListing" element={<ProductListing />} />
           <Route path="/Login" element={<Login />} />
           <Route path="/SignUp" element={<SignUp />} />
-          <Route path="/myCart" element={<Cart />} />
-          <Route path="/Wishlist" element={<Wishlist />} />
+          <Route path="/myCart" element={privateRoute ? <Cart /> : <Login />} />
```

Comment:
Suggestion: If you are using PascalCase use it for every route path to maintain consistency.

Reply:
noted

Reply:
done